docs/resources/aom_alarm_rule.md

```diff
@@ -0,0 +1,122 @@
+  The default value is **2**.
+
+* `alarm_actions` - (Optional, List, ForceNew) Specifies the action triggered by an alarm. This is a list of strings.
+  Changing this creates a new resource.
+
+* `alarm_action_enabled` - (Optional, Bool, ForceNew) Specifies whether to enable the action to be triggered by an alarm.
+  The default value is true. Changing this creates a new resource.
+
+* `ok_actions` - (Optional, List, ForceNew) Specifies the action triggered by the clearing of an alarm.
+  This is a list of strings. Changing this creates a new resource.
+
+* `insufficient_data_actions` - (Optional, List, ForceNew) Specifies the action triggered when the data is not enough.
+  This is a list of strings. Changing this creates a new resource.
+
+* `metric_name` - (Required, string, ForceNew) Specifies the alarm metric name. Changing this creates a new resource.
+
+* `namespace` - (Required, String, ForceNew) Specifies the alarm namespace. Changing this creates a new resource.
+
+* `period` - (Required, Int) Specifies the alarm checking period in milliseconds.
+  The value can be **60000**, **300000**, **900000** and **3600000**.
+
+* `statistic` - (Required, String, ForceNew) Specifies the data rollup methods. The value can be **maximum**,
+  **minimum**, **average**, **sum** and **sampleCount**. Changing this creates a new resource.
+
+* `comparison_operator` - (Required, String) Specifies the comparison condition of alarm thresholds.
+  The value can be **>**, **=**, **<**, **>=** or **<=**.
+
+* `threshold` - (Required, Int) Specifies the alarm threshold.
+
+* `unit` - (Required, String, ForceNew) Specifies the data unit. Changing this creates a new resource.
+
+* `evaluation_periods` - (Required, Int) Specifies the alarm checking evaluation periods.
+  The value can be **1**, **2**, **3**, **4** and **5**.
+
+* `dimensions` - (Optional, List, ForceNew) Specifies the list of metric dimensions. The structure is described below.
+  Changing this creates a new resource.
+
+The `dimensions` block supports:
+
+* `name` - (Required, String, ForceNew) Specifies the dimension name. Changing this creates a new resource.
+
+* `value` - (Required, String, ForceNew) Specifies the dimension value. Changing this creates a new resource.
+
+-> **NOTE:** You can get more information about `metric_name`, `namespace`, `unit` and `dimensions`
+  from [Metric Overview](https://support.huaweicloud.com/intl/en-us/productdesc-aom/aom_06_0014.html).
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - Indicates the alarm rule ID.
+
+* `state_value` - Indicates the alarm status.
+
+* `state_reason` - Indicates the reason of alarm status.
+
+## Timeouts
+
+This resource provides the following timeouts configuration options:
+
+* `create` - Default is 5 minute.
+* `update` - Default is 5 minute.
+* `delete` - Default is 5 minute.
+
+## Import
+
+AOM alarm rules can be imported using the `id`, e.g.
```

Comment:
missing ending character, such as `:` or `,`
a. `AOM alarm rules can be imported using the id, e.g.:`
b. `AOM alarm rules can be imported using the id, e.g.,`

Reply:
no ending character is right